package.json

```
@@ -33,6 +33,7 @@
   "dependencies": {
     "@tanstack/react-query": "^5.84.2",
     "axios": "^1.11.0",
+    "framer-motion": "^12.23.12",
```

Biome (2.1.2)

[error] The key framer-motion was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)